extensions/amp-story/1.0/amp-story-tooltip.css

```diff
@@ -112,13 +101,15 @@
   position: absolute !important;
   box-sizing: border-box !important;
   text-decoration: none !important;
+  text-shadow: none !important;
+  font-weight: 500 !important;
   display: flex !important;
   justify-content: space-between !important;
   align-items: center !important;
   background: white !important;
-  box-shadow: 0 1px 3px 1px rgba(0,0,0,0.24) !important;
+  box-shadow: 0px 6px 16px rgba(0,0,0,0.16) !important;
```

Comment:
Super nitty: not a big deal since we haven't really been following these guidelines, but I think for "floating buttons" the material guidelines recommend 12px instead of 16

Reply:
tried 12 and 16 both and 16 stands out much better on phone screen.